src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -179,12 +179,14 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	private _labelComputer?: TerminalLabelComputer;
 	private _userHome?: string;
 	private _hasScrollBar?: boolean;
+	private _target?: TerminalLocation | undefined;
 
-	get target(): TerminalLocation | undefined { return this.xterm?.target; }
+	get target(): TerminalLocation | undefined { return this._target; }
```

Comment:
not sure why it won't let me do

```
this.xterm?.target = value 
```

Reply:
That could end up resolving to `undefined = value` which isn't valid, afaik it's just not covered by that feature, need to do the if